.ci/actions/os-preparation/action.yml

```diff
@@ -0,0 +1,44 @@
+name: 'OS Preparation'
+description: 'Prepares the Operating Systems array'
+
+inputs:
+  repository:
+    description: "The repository containing the matrix-os.json os configuration"
+    required: true
+  branch:
+    description: "The repository branch from which the matrix configuration must be taken"
+    required: false
+    default: "main"
+
+outputs:
+  operating-systems:
+    description: Configured job operating systems
+    value: ${{ steps.set-os.outputs.operating_systems }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check out configs repo
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ inputs.repository }}
+        ref: ${{ inputs.branch }}
+        path: matrix_preparation
+    - id: set-os
+      env:
+        repo_name: ${{ github.event.repository.name }}
+        event: ${{ github.event_name }}
+        label_name: ${{ github.event.label.name }}
+        config_file: matrix_preparation/.github/matrix-os.json
+      shell: bash
+      run: |
+        os=$(jq --arg config_file "$config_file" --arg repo "$repo_name" --arg event "$event" --arg event_value "$label_name" 'map(.| select(.event==$event and ((.value==null and ((.repository | index( $repo))) or (.repository==null)) or .value==$event_value)))' $GITHUB_WORKSPACE/$config_file | jq "if (length > 1) then (.[] | select(.repository!=null)) else (.[] | select(.repository==null)) end | .os")
+        if [ -z "$os" ]; then 
+          echo "No configuration found for [repo=$repo_name, event=$event, value=$label_name] at $GITHUB_WORKSPACE/$config_file"
+          exit 1
+        else 
+          echo "::set-output name=operating_systems::$(echo $os)" 
```

Comment:
this was already deprecated and fixed in https://issues.redhat.com/browse/BXMSPROD-1937

Reply:
you are right, I missed to propagate the change here. Doing it right now :)